modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java

```diff
@@ -487,6 +464,51 @@ public PortletURL getPortletURL() throws PortalException {
 		return portletURL;
 	}
 
+	public SearchContainer<CommerceOrder> getSearchContainer()
+		throws PortalException {
+
+		if (_searchContainer != null) {
+			return _searchContainer;
+		}
+
+		_searchContainer = new SearchContainer<>(
+			_cpRequestHelper.getLiferayPortletRequest(), getPortletURL(), null,
+			"no-orders-were-found");
+
+		List<CommerceOrder> commerceOrders;
+		long commerceOrdersTotal;
+
+		String keywords = ParamUtil.getString(_httpServletRequest, "keywords");
+
+		if (isOpenOrderContentPortlet()) {
+			commerceOrders = _commerceOrderService.getUserPendingCommerceOrders(
+				_cpRequestHelper.getCompanyId(),
+				_cpRequestHelper.getChannelGroupId(), keywords,
+				_searchContainer.getStart(), _searchContainer.getEnd());
+
+			commerceOrdersTotal =
+				_commerceOrderService.getUserPendingCommerceOrdersCount(
+					_cpRequestHelper.getCompanyId(),
+					_cpRequestHelper.getChannelGroupId(), keywords);
+		}
+		else {
+			commerceOrders = _commerceOrderService.getUserPlacedCommerceOrders(
+				_cpRequestHelper.getCompanyId(),
+				_cpRequestHelper.getChannelGroupId(), keywords,
+				_searchContainer.getStart(), _searchContainer.getEnd());
+
+			commerceOrdersTotal =
+				_commerceOrderService.getUserPlacedCommerceOrdersCount(
+					_cpRequestHelper.getCompanyId(),
+					_cpRequestHelper.getChannelGroupId(), keywords);
+		}
+
+		_searchContainer.setTotal((int)commerceOrdersTotal);
+		_searchContainer.setResults(commerceOrders);
```

Comment:
I think we sort this.
```suggestion
		_searchContainer.setResults(commerceOrders);
		_searchContainer.setTotal((int)commerceOrdersTotal);
```